docs/lake/design.md

```diff
@@ -279,8 +296,38 @@ Note that the data in commits `<tag-1>`, `<tag-2>`, and `<tag-3>` remains
 in the pool and scans can be performed on older snapshots of the pool
 as long as the data is not deleted.
 
+When multiple commits are given to commit, they are automatically squashed
+into a new commit.  The old messsage fields are lost and must be replaced
+by a new message.  Since this is typically driven with automation
+we do not yet have an edit cycle like `git commit` offers to merge squashed
+commit messages into the new messages via an editor.
+
+A squash may be separately executed without a journal commit using the
+`zed lake squash` command, e.g.,
+```
+zed lake squash -p logs <tag-1> <tag-2> <tag-3>
+(merged commit <squash-tag> printed to stdout)
+zed lake delete -p logs <tag-1> <tag-2> <tag-3>
+(delete commit <del-tag> printed to stdout)
+zed lake commit -p logs <squash-tag> <del-tag>
+```
+
+Note that delete can be used separately from squash to delete entire commits
+or individual segments at any time.  This is handy when importing data by
+mistake:
+```
+zed lake load -p logs oops.ndjson
+(commit <tag> etc printed to stdout)
+zed lake delete -p <tag>
+```
+In this case, the data will be deleted from any subsequent scans but still
+exists in the lake and can be accessed via time travel.
```

Comment:
````suggestion
zed lake load -p logs oops.ndjson
(commit <tag> etc printed to stdout)
zed lake delete -p logs <tag>
```
In this case, the data will be deleted from any subsequent scans but still
exists in the lake and can be accessed via time travel.
````

In addition to the small correction I'm suggesting here, just to make sure I understand overall... is it to illustrate the "...but still exists..." point that you didn't include the additional step of committing the `<del-tag>` that would have been returned by the last command?

Reply:
Good catch.  Delete has a `-commit` flag so I will add that.